bin/git-ranch

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+echo "Howdy pardner! Looks like you've mistaken a git command. Did you mean \`git branch\`? See \`git --help\`. Yee-haw!
+
+While yer in the ranchin' mood, why don't ya try this recipe for bacon 'n beans?
+
+Ingredients
+
+1 pound 90% lean ground beef*
+1 pound bacon
+1 medium yellow onion, chopped
+2 (28 ounce) cans baked beans (I used Bush's Original)
+1 (15 ounce) can pork and beans
+2 (15 ounce) cans black beans, drained and rinsed
+1 cup barbecue sauce (I used Sweet Baby Ray's Originial)
```

Comment:
I'd like to see an option for diabetics here as well. Most barbecue sauce contains a lot of sugar!